fabric-biomes-v1/src/main/java/net/fabricmc/fabric/mixin/biomes/SpawnBiomesMixin.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.biomes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.fabricmc.fabric.impl.biomes.BiomeLists;
+import net.minecraft.world.biome.Biome;
+import net.minecraft.world.biome.source.BiomeSource;
+
+@Mixin(BiomeSource.class)
+public class SpawnBiomesMixin 
+{
+	@Shadow
+	@Final
+	private static List<Biome> SPAWN_BIOMES;
+	
+	@Inject(at = @At("HEAD"), cancellable = true, method = "getSpawnBiomes")
+	private void getSpawnBiomes(CallbackInfoReturnable<List<Biome>> info)
+	{
+		List<Biome> toReturn = new ArrayList<>();
+		for (Biome biome : SPAWN_BIOMES)
+			toReturn.add(biome);
```

Comment:
Iteration of `SPAWN_BIOMES` can be removed, and `new ArrayList<>()` replaced with `new ArrayList<>(SPAWN_BIOMES)`